src/bufr/BufrParser/Exports/Splits/CategorySplit.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#pragma once
+
+#include "Split.h"
+
+#include <string>
+#include <vector>
+#include <map>
+
+
+namespace Ingester
+{
+    /// \brief Data splitter class that splits data according to a predefined categories.
```

Comment:
May be add an example of one or two categories.

Reply:
Added extra documentation to explain things with more detail.